airbyte-integrations/connectors/source-singlestore/src/main/resources/spec.json

```diff
@@ -0,0 +1,202 @@
+        "type" : "string",
+        "order" : 5
+      },
+      "ssl_mode" : {
+        "title" : "SSL modes",
+        "description" : "SSL connection modes.",
+        "type" : "object",
+        "order" : 7,
+        "oneOf" : [
+          {
+            "title" : "disable",
+            "description" : "Do not use SSL/TLS",
+            "required" : [
+              "mode"
+            ],
+            "properties" : {
+              "mode" : {
+                "type" : "string",
+                "const" : "disable",
+                "order" : 0
+              }
+            }
+          },
+          {
+            "title" : "required",
+            "description" : "Only use SSL/TLS for encryption. Do not perform certificate or hostname verification. This mode is not safe for production applications.",
+            "required" : [
+              "mode"
+            ],
+            "properties" : {
+              "mode" : {
+                "type" : "string",
+                "const" : "required",
+                "order" : 0
+              }
+            }
+          },
+          {
+            "title" : "Verify CA",
+            "description" : "Use SSL/TLS for encryption and perform certificates verification, but do not perform hostname verification.",
+            "required" : [
+              "mode",
+              "ca_certificate"
+            ],
+            "properties" : {
+              "mode" : {
+                "type" : "string",
+                "const" : "verify-ca",
+                "order" : 0
+              },
+              "ca_certificate" : {
+                "type" : "string",
+                "title" : "CA certificate",
+                "description" : "CA certificate",
+                "airbyte_secret" : true,
+                "multiline" : true,
+                "order" : 1
+              },
+              "client_certificate" : {
+                "type" : "string",
+                "title" : "Client certificate",
+                "description" : "Client certificate (this is not a required field, but if you want to use it, you will need to add the <b>Client key</b> as well)",
+                "airbyte_secret" : true,
+                "multiline" : true,
+                "order" : 2,
+                "always_show" : true
+              },
+              "client_key" : {
+                "type" : "string",
+                "title" : "Client key",
+                "description" : "Client key (this is not a required field, but if you want to use it, you will need to add the <b>Client certificate</b> as well)",
```

Comment:
If I remember correctly, then this parameter should be specified as a path to the file.
So I have 2 questions:
 - Is it possible to place this file in the airbyte?
 - Let's add information about the parameter format and examples here (and also for other parameters as well).

Reply:
It's not a file path. It's a private key that could be imported to generated keystore with certificate. Format of inputs:
**certificate pem string:**
-----BEGIN CERTIFICATE-----
MIIDfTCCAmWgAwIBAgIUDQhA/TESPTo8FqaDnwWfK4U7bBUwDQYJKoZIhvcNAQEL
*****
sstAtCcXmRVKjAvYS9wgmhIA6kl6T5qQW0e1VL3TVrQP
-----END CERTIFICATE-----

**Client key string:**
-----BEGIN PRIVATE KEY-----
MIIEvwIBADANBgkqhkiG9w0BAQEFAASCBKkwggSlAgEAAoIBAQDDn8c2y8dV8eDs
*****
iO7tZjy8g3wauEibjehS4YmaTg==
-----END PRIVATE KEY-----

After they are added to key store, the key store path is passed to jdbc connection string parameter with it's password.

Reply:
I'll add examples to documentation (PLAT-7037 added comment). Is it ok?